docs/tutorial_basics.rst

```diff
@@ -144,26 +147,32 @@ created containing the NLU model.
 So far, we've got an NLU model, a domain defining the actions our bot can
 take, and inputs it should handle (intents & entities). We are still
 missing the central piece, **stories to tell our bot what to do at which
-point in the dialogue**. There are two different ways to create stories (and
+point in the dialogue**.
+
+A **story** is a training data sample for the dialogue system.
+There are two different ways to create stories (and
 you can mix them):
 
 - create the stories by hand, writing them directly to a file
 - create stories using interactive learning (see :ref:`tutorial_interactive_learning`).
 
 For this example, we are going to create the stories by writing them directly
-to ``stories.md``. But be aware, although it is a bit faster to write
-stories directly by hand instead of using interactive learning, special
-care needs to be taken when using slots, as they need to be properly set in the
-stories. But enough talking, let's head over to our stories:
+to ``stories.md``.
+Stories begin with ``##`` and a string as an identifier. User actions start
+with an asterisk, and bot actions are specified by lines beginning with a
+dash. The end of a story is denoted by a newline. See :ref:`stories` for
+more information about the data format.
+
+Enough talking, let's head over to our stories:
 
 .. literalinclude:: ../examples/moodbot/data/stories.md
     :linenos:
     :language: md
 
-Stories begin with ``##`` and a string as an identifier. User actions start
-with an asterisk, and bot actions are specified by lines beginning with a
-dash. The end of a story is denoted by a newline. See :ref:`stories` for
-more information about the data format.
+Be aware, although it is a bit faster to write
+stories directly by hand instead of using interactive learning, special
+care needs to be taken when using slots, as they need to be properly set in the
```

Comment:
unless they're unfeaturized slots - idk if that's worth mentioning here or if its enough later in the detailed slots section

Reply:
I think it is better to leave it to the Slots section. The reader has very vague idea of what a slot is right now, not speaking about the type